### ISSUE TITLE
This repository is no longer maintained and will not be open-sourced

### ISSUE THREAD

**Issue by fabien-joubert** [NativeDocuments/docx-wasm-client-side#7]
# This repository is no longer maintained

Since the last commit was over 2 years ago. ([see the last commit](https://github.com/NativeDocuments/docx-wasm-client-side/commit/4e4181fd309b6cf9074ddafa452b56c5aeba125c))
By the way, there has only been one commit since the initial release. ([see the commit history](https://github.com/NativeDocuments/docx-wasm-client-side/commits/master))

_These two commits were done on the same day._

# This repository will NOT be open-sourced

See the issue #4 in particular [this comment](https://github.com/NativeDocuments/docx-wasm-client-side/issues/4#issuecomment-590537852).

In short: **This repository will NOT be open-sourced**.

# And now ?

Now **I open this issue to discuss alternatives and other solutions**.

1. Do you know of other ways to convert `docx` file to `pdf` client side?
2. Do you know of an alternative library?

Open source and maintained solutions will be appreciated. However feel free to give any suitable solution.
- - - -
Any help would be appreciated.
This will help anyone in my case who will come across this repository.

_Thank you in advance for your future contributions._

**Comment by janswist** [comment]
I second that question. NativeDocument's solution seemed the best and they suddenly disappeared.

**Comment by silvium** [comment]
This s#!7 projects pollutes google and all tutorial sites and stuff. I wasted days searching and finally found https://github.com/zVolodymyr/docxjs Now, it's not docx to pdf but i'm pretty sure there's a way to convert the HTML it outputs to canvas, then PDF. mammoth.js has terrible accuracy issues. https://github.com/zVolodymyr/docxjs is not perfect either but it's the best I saw, except for the "Office Editing for Docs, Sheets & Slides" extension from the chrome web store (it has a minified JS that i'm not sure I'd be allowed to use even if I had the time to figure out its methods)

**Comment by shubhendusaurabh** [comment]
for folks to coming to this page may want to check out https://github.com/allotropia/zetajs a wasm port of libreoffice
